File: TypeScript/UberX.ts
import { Account } from "./Account";
import { Car } from "./Car";

export class UberX extends Car {
    brand: String;
    model: String;

    constructor( license: String, driver: Account, brand: String, model: String ) {
        super( license, driver );
        this.brand = brand;
        this.model = model;
    }

    public printDataCar(): void {
        super.printDataCar();
        console.log('Model: ' + this.model + ' Brand: ' + this.brand);
    }
}

File: Python/uberX.py
from car import Car


class UberX(Car):
    __brand = str
    __model = str

    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.__brand = brand
        self.__model = model

    def printDataCar(self):
        super().printDataCar()
        print('Model: ', self.__model, 'Brand: ', self.__brand)
